lld/MachO/UnwindInfoSection.h

```diff
@@ -34,6 +34,17 @@ class UnwindInfoSection : public SyntheticSection {
 
 UnwindInfoSection *makeUnwindInfoSection();
 
+// LLD's internal representation of a compact unwind entry.
+struct CompactUnwindEntry {
+  uint64_t functionAddress;
+  uint32_t functionLength;
+  compact_unwind_encoding_t encoding;
+  Symbol *personality;
+  InputSection *lsda;
+
+  void relocateOneCompactUnwindEntry(const Defined *d);
```

Comment:
I wish this function could return `std::optional<CompactUnwindEntry>`, but it looks like we need to set `functionAddress` even if there is no unwind entry (I wonder if this really is required). 

At the very least, can you add a comment explaining what this function does?